### PR TITLE
Update/start client guide tool

### PR DESCRIPTION
**Pull request by gene-zhan**
merge update

### CODE REVIEW

File: docs/develop/guides/reentrancy-guard/img.png
Comment (dumdumgoose) [inline docs/develop/guides/reentrancy-guard/img.png:1]: Try to use `.svg` picture as much as you can, for snippet like this, you can copy the text to https://carbon.now.sh/ and export an `svg` file

**Review by dumdumgoose — CHANGES_REQUESTED** [review]

**Comment by dumbeng** [comment]
LGTM

**Review by dumbeng — APPROVED** [review]
LGTM

**Review by luketheart — APPROVED** [review]
LGTM

**Pull request merged**